.eslintrc.js

```diff
@@ -387,6 +391,6 @@ module.exports = {
     '@typescript-eslint/no-parameter-properties': 'error',
     '@typescript-eslint/no-var-requires': 'error',
     '@typescript-eslint/prefer-namespace-keyword': 'error',
-    '@typescript-eslint/type-annotation-spacing': 'error'
+    '@typescript-eslint/type-annotation-spacing': 'error',
```

Comment:
you can leave out this comma. Usually the rule `comma-dangle` should complain here. But since we don't lint this file, this is not really an issue and not complained by the linter. But just to be consistent